proto/src/constants.rs

```diff
@@ -46,6 +46,30 @@ pub const DEFAULT_REPLICATION_ORIGIN: &str = "repl://localhost:8444";
 /// Default replication poll window in seconds.
 pub const DEFAULT_REPL_TASK_POLL_INTERVAL: u64 = 15;
 
+/// Minimum replication poll window in seconds:
+pub const MIN_REPL_TASK_POLL_INTERVAL: u64 = 3;
+
+// Estimate of max write ops that can be processed in the corresponding replication poll window
+pub const MAX_WRITE_THROUGHPUT_BY_TASK_POLL_INTERVAL: [(u64, u64); 13] = [
+    (2945, 3),
+    (4081, 4),
+    (5294, 5),
+    (6351, 6),
+    (7408, 7),
+    (8280, 8),
```

Comment:
Given this appears linear, it could be a function instead of a constant.

Reply:
How was this calculated, surely it's heavily dependent on system architecture etc?

Reply:
> How was this calculated, surely it's heavily dependent on system architecture etc?

The numbers come from our benchmarks, but you're right in saying they are ... not consistent across different machines. But I have another secret plan to fix that ;-). For now I think it makes sense to just assume they're correct and polish everything else.

Reply:
> Given this appears linear, it could be a function instead of a constant.

Sure, but I think we first need the big chonky benchmarks we were discussing yesterday before interpolating the points. Also we wouldn't need to compute weird indexes as we're currently doing, it would be much more elegant!

Reply:
>  For now I think it makes sense to just assume they're correct and polish everything else.

I'll have to disagree on that one, for now.

Reply:
As do I.